dupedetection/README.md

```diff
@@ -32,14 +32,14 @@ Then create the following folder structure:
 
 ```
 mkdir -p /home/$USER/pastel_dupe_detection_service/dupe_detection_input_files/
-mkdir -p /home/$USER/pastel_dupe_detection_service/dupe_detection_support_files/
+mkdir -p /home/$USER/pastel_dupe_detection_service/support_files/
 mkdir -p /home/$USER/pastel_dupe_detection_service/dupe_detection_output_files/
 mkdir -p /home/$USER/pastel_dupe_detection_service/dupe_detection_processed_files/
 mkdir -p /home/$USER/pastel_dupe_detection_service/dupe_detection_rare_on_internet/
 mkdir -p /home/$USER/pastel_dupe_detection_service/mobilenet_v2_140_224/
```

Comment:
could you remove prefix `dupe_detection` of other folder? like:
/dupe_detection_processed_files,
dupe_detection_rare_on_internet
dupe_detection_input_files

Reply:
not needed yet